--- wagtail/users/utils.py.orig	2023-03-03 10:00:20
+++ wagtail/users/utils.py	2023-03-03 10:00:31
@@ -40,7 +40,7 @@
 
     gravatar_url = "{gravatar_provider_url}/{hash}?{params}".format(
         gravatar_provider_url=gravatar_provider_url.rstrip("/"),
-        hash=hashlib.md5(email.lower().encode("utf-8")).hexdigest(),
+        hash=hashlib.md5(email.lower().encode("utf-8"), usedforsecurity=False).hexdigest(),
         params=urlencode({"s": size, "d": default}),
     )
 
